diff --git a/packages/backend/src/const.ts b/packages/backend/src/const.ts
index 1ca0397206..2f768c476e 100644
--- a/packages/backend/src/const.ts
+++ b/packages/backend/src/const.ts
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: AGPL-3.0-only
  */

-export const MAX_NOTE_TEXT_LENGTH = 3000;
+export const MAX_NOTE_TEXT_LENGTH = 5000;

 export const USER_ONLINE_THRESHOLD = 1000 * 60 * 10; // 10min
 export const USER_ACTIVE_THRESHOLD = 1000 * 60 * 60 * 24 * 3; // 3days
